deploy/twingate-operator/tests/values.test.yaml

```diff
@@ -0,0 +1,69 @@
+twingateOperator:
+  apiKey: "<api key>"
+  network: "<network slug>"
+  remoteNetworkId: "<remote network id>"
+  logFormat: "plain"
+  logVerbosity: "verbose"
+  defaultResourceTags:
+    cluster: "test-cluster"
+    owner: "eran"
+
+kubernetes-access-gateway:
+  enabled: false
+
+image:
+  repository: twingate/kubernetes-operator
+  pullPolicy: IfNotPresent
+  tag: ""
+
+imagePullSecrets: []
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: true
+  annotations: {}
+  name: ""
```

Comment:
It's unclear to me when I should add to this `values.test.yaml` to be captured by the snapshot test. These are default values. Some other values e.g. `podAnnotations` or `podLabels` are not. I think the snapshot test should test the default `values.yaml` file. Other custom fields like `podAnnotations` should have be tested on its own in the `deployment_test.yaml` because they only affects the deployment template.

Reply:
I can see why it might be unclear. The reason I added `values.test.yaml` is so we don't have to keep adding the 3 required fields (`network`, `apiKey` and `remoteNetwork`) in `twingateOperator` for other test suites repeatedly. I updated the setup ([80a85d8](https://github.com/Twingate/kubernetes-operator/pull/728/commits/80a85d82721306451ff5827a4807242a88ced1c6)) to set the `twingateOperator` value at the test suite level instead.

Reply:
If that's the issue, why do we need other values than those 3 required fields (`network`, `apiKey` and `remoteNetwork`) in the `values.test.yaml`?